.golangci.yml

```diff
@@ -29,7 +29,9 @@ linters:
     # - gochecknoinits
     - gocritic
     - gocyclo
+    - goconst
     - gofmt
+    - goheader
```

Comment:
We have https://github.com/submariner-io/submariner/blob/master/.github/workflows/linting.yml#L53-L61 to check for legal headers. Maybe we should do a comparison of the two and decide which we want to enable?

Reply:
Yeah, may be the other is fine already?

Reply:
yes. I was just thinking to keep all go linters together. However, goheaders is acting wierd and throwing an error which I can't figure out.

`pkg/routeagent_driver/cleanup/cleanup.go:13:31: Missed string: (goheader)
See the License for the specific language governing permissions and
                              ^
pkg/routeagent_driver/constants/constants.go:13:31: Missed string: (goheader)
See the License for the specific language governing permissions and
                              ^
pkg/routeagent_driver/environment/env.go:13:31: Missed string: (goheader)
See the License for the specific language governing permissions and
                              ^
make: *** [/opt/shipyard/Makefile.inc:65: golangci-lint] Error 1
[submariner]$ make golangci-lint
time="2021-02-09T10:40:59Z" level=fatal msg="exit status 2"
make: *** [golangci-lint] Error 1
Makefile.dapper:14: recipe for target 'golangci-lint' failed
Error: Process completed with exit code 2.`

Reverting to using GitHub workflows for license checks.